cmd/create/oidcconfig/cmd.go

```diff
@@ -310,6 +326,11 @@ func (s *CreateOidcConfigAutoStrategy) execute(r *rosa.Runtime) {
 			"to S3 bucket '%s': %s", bucketName, err)
 		os.Exit(1)
 	}
+	secretsARN, err := r.AWSClient.CreateSecretInSecretsManager(privateKeySecretName, string(privateKey[:]))
```

Comment:
Changing so that if the bucket name is still not valid for some reason we don't create the secret